api/fixtures/activity-progress-labels.yml

```diff
@@ -0,0 +1,26 @@
+App\Entity\ActivityProgressLabel:
+  activityProgressLabel1:
+    camp: '@camp1'
+    position: 1
+    title: <word()>
```

Comment:
i would use a fix string here, this makes it easier to follow when a test fails. (specially since they are ordered)